documentation/staging/content/userguide/prerequisites/introduction.md

```diff
@@ -15,9 +15,11 @@ For the current production release {{< latestVersion >}}:
 * For domain home source type `Model in Image`, WebLogic Deploy Tooling 1.9.11+.
 * Either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930, Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0.
    * The existing WebLogic Server image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`,
-   has all the necessary patches applied.
-   * Check the WLS version with `docker run container-registry.oracle.com/middleware/weblogic:12.2.1.3 sh -c` `'source $ORACLE_HOME/wlserver/server/bin/setWLSEnv.sh > /dev/null 2>&1 && java weblogic.version'`.
-   * Check the WLS patches with `docker run container-registry.oracle.com/middleware/weblogic:12.2.1.3 sh -c` `'$ORACLE_HOME/OPatch/opatch lspatches'`.
+   has this patch applied.
+   {{% notice warning %}}
+   General Availability (GA) images are suitable for demonstration and development purposes _only_ where the environments are not available from the public Internet; they are **not acceptable for production use**. In production, you should always use CPU (patched) images from [OCR]({{< relref "/userguide/base-images/ocr-images.md" >}}) or create your images using the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/userguide/tools/create-image/) (WIT) with the `--recommendedPatches` option. For more guidance, see [Apply the Latest Patches and Updates](https://www.oracle.com/pls/topic/lookup?ctx=en/middleware/standalone/weblogic-server/14.1.1.0&id=LOCKD-GUID-2DA84185-46BA-4D7A-80D2-9D577A4E8DE2) in _Securing a Production Environment for Oracle WebLogic Server_. For details on how to obtain or create the image, see [WebLogic images]({{< relref "/userguide/base-images/_index.md" >}}).
+   {{% /notice %}}
+   * Check the WLS version and patches with `imagetool inspect --image=container-registry.oracle.com/middleware/weblogic:12.2.1.3 --patches`.
```

Comment:
IMO it is not appropriate to remove the 'generic' advice here for using the docker command, since getting the image tool to work requires several additional steps. Also note that the raw reference to 'imagetool' leaves new users at a loss, since they may not realize this is the WIT. 

Could we instead keep the original advice, and add a link to the "inspect" guidance in the domain images chapter?  Plus double check that the inspect guidance in the images chapter provides enough context to define "WIT" and where to get instructions for installing same...

Reply:
@ddsharpe

Reply:
We are encouraging customers to build thir images using the image tool and I think we need to be consistent.  I am fine adding "Check the WLS version and patches using the WebLogic Image Tool `inspect` command ..."  and add a link to the WIT inspect docs.

Reply:
I solved this according to the comment from @mriccell If this is not correct, please let me know